src/Host.cpp

```diff
@@ -907,6 +907,12 @@ bool Host::checkForMappingScript()
     return ret;
 }
 
+bool Host::checkForSpeedwalkScript()
```

Comment:
I'd say checkForCustomSpeedwalk, since we don't have the idea of a "speedwalk script", just a "mapping script".